scripts/src/workflowtesting/checkprforci.py

```diff
@@ -41,9 +45,13 @@ def check_if_ci_only_is_modified(api_url):
                 workflow_found = True
             elif pattern_test.match(filename):
                 workflow_found = True
+            elif pattern_release.match(filename) or pattern_readme.match(filename) or pattern_docs.match(filename):
```

Comment:
I would prefer putting these patterns in an array and iterating on them instead of numerating, e.g. `workflow_files = [r".github/workflows/.*", r"scripts/.*", r"tests/.*"]` and `skip_build_files = [r"release/release_info.json", r"README.md", r"docs/([\w-]+)\.md"]`. Then iterate on them in the while loop instead of repeating. Please adjust the names to your preference since I couldn't come up with a good name.

ref: https://github.com/openshift-helm-charts/charts/pull/332/files#r711108836

Reply:
I think this could be part of the refactoring when we get to it.